tests/bulk_create/models.py

```diff
@@ -51,3 +53,17 @@ class TwoFields(models.Model):
 
 class NoFields(models.Model):
     pass
+
+
+class NullableFields(models.Model):
```

Comment:
I thought that I need to check that each sql-function exists and works correctly. I agree that I did more than needed.
Right now `BulkInsertMapper` has 7 conversion functions:
```
    BLOB = 'TO_BLOB(%s)'
    CHAR = 'TO_CHAR(%s)'
    DATE = 'TO_DATE(%s)'
    NCHAR = 'TO_NCHAR(%s)'
    NCLOB = 'TO_NCLOB(%s)'
    NUMBER = 'TO_NUMBER(%s)'
    TIMESTAMP = 'TO_TIMESTAMP(%s)'
```
So, do I need to leave only 7 fields in `class NullableFields`? Which fields would be better to remove?